README.md

```diff
@@ -29,7 +29,8 @@ Run `ng test` to execute the unit tests via [Karma](https://karma-runner.github.
 
 Run `ng e2e` to execute the end-to-end tests via [Protractor](http://www.protractortest.org/).
 
-## Further help
+## Docker Setup
 
-This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 1.6.5.
-To get more help on the Angular CLI use `ng help` or go check out the [Angular CLI README](https://github.com/angular/angular-cli/blob/master/README.md).
+git clone https://github.com/Cloud-CV/EvalAI-ngx.git && cd EvalAI-ngx
```

Comment:
@anweshknayak please refer steps from here https://github.com/Cloud-CV/EvalAI/blob/master/README.md#using-docker
Write down in this way.